### PR TITLE
Added guiwrapper for fcitx-qt5

### PR DESCRIPTION
**Pull request by xuzhao9**
Still not sure whether the quotes are required in "QPlatformInputContextFactoryInterface_iid".

### CODE REVIEW

File: guiwrapper/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  PROPERTIES VERSION 1.0
+  AUTOMOC TRUE)
+
+target_link_libraries(fcitx-qt5-gui-wrapper
+      Qt5::Core
+      Qt5::Gui
+      Qt5::Widgets
+      FcitxQt5::DBusAddons
+      FcitxQt5::WidgetsAddons
+       ${FCITX4_FCITX_UTILS_LIBRARIES}
```
Comment (wengxt) [inline guiwrapper/CMakeLists.txt]: extra space

File: guiwrapper/common.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of         *
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the          *
+ *  GNU General Public License for more details.                           *
+ *                                                                         *
+ *  You should have received a copy of the GNU General Public License      *
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.  *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef _FCITX_TOOL_GUI_COMMON_H
```
Comment (wengxt) [inline guiwrapper/common.h]: change the macro

File: guiwrapper/common.h [hunk 3]
```diff
@@ -0,0 +1,27 @@
+/***************************************************************************
+ *   Copyright (C) 2012~2012 by CSSlayer                                   *
```
Comment (wengxt) [inline guiwrapper/common.h]: add yourself

File: platforminputcontext/main.h [hunk 4]
```diff
@@ -29,7 +29,7 @@ class QFcitxPlatformInputContextPlugin : public QPlatformInputContextPlugin
 {
     Q_OBJECT
 public:
-    Q_PLUGIN_METADATA(IID QPlatformInputContextFactoryInterface_iid FILE "fcitx.json")
+    Q_PLUGIN_METADATA(IID "QPlatformInputContextFactoryInterface_iid" FILE "fcitx.json")
```
Comment (wengxt) [inline platforminputcontext/main.h]: this is not right. This is actually a macro in ./QtGui/5.7.1/QtGui/qpa/qplatforminputcontextplugin_p.h

**Comment by xuzhao9** [comment]
Changed according to csslayer's review.

**Pull request merged**